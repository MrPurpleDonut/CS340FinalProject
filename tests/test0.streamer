num two;
two = 2;
