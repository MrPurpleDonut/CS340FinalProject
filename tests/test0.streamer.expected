Program parsed correctly!
---------------------------
--------Printing AST-------
---------------------------
Program
    Function List
    Statement List
        Declaration statement: two
            Type: num
        Assignment statement: two
            Number: 2.0
